src/styles/StyleUtils.js

```diff
@@ -637,18 +637,18 @@ function getLoginPagePromoStyle() {
  * Generate the styles for the ReportActionItem wrapper view.
  *
  * @param {Boolean} [isHovered]
- * @param {Boolean} [isLoading]
+ * @param {Boolean} [hasError]
```

Comment:
I'm not sure I understand this change? `isLoading` makes sense, no?

Reply:
Yes, I changed the name because [we are nuking](https://github.com/Expensify/App/pull/25557#discussion_r1299839956) `action.isLoading`, and this use had to be adjusted.